src/App.js

```diff
@@ -1,25 +1,159 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import "./SlotMachine.css";
+
+const symbols = ["🍍", "🐠", "🥥", "🌴", "🐒", "🏴‍☠️"];
+const credits = [1, 5, 10, 20, 50, 100];
+
+function getRandomSymbol() {
+  return symbols[Math.floor(Math.random() * symbols.length)];
+}
+
+const SlotMachine = () => {
+  const [reels, setReels] = useState(
+    Array(3)
```

Comment:
instead of directly initializing this state with prefills, you can wrap this into a method like getInitialReelPlaceholders or descriptive that makes sense to the reader on first read. When I look at this, it does take me a little mind computation trying to figure out what it is doing.